Makefile

```diff
@@ -142,10 +142,9 @@ latexdoc texdoc doc:
 	(cd mathics/doc/latex && $(MAKE) doc)
 
 #: Build JSON ASCII to unicode opcode table and operator table
-mathics/data/op-tables.json mathics/data/operators.json:
+mathics/data/operator-tables.json mathics/data/op-tables.json mathics/data/operators.json:
```

Comment:
operators.json is not in mathics-core/mathics/data but in mathics_scanner/data/

Reply:
(I should have put a line in `mathics_scanner.generate.build_operator_tables` that prints where `operators.json` is stored...